nursery/authenticate-message-hash-based.yml

```diff
@@ -0,0 +1,20 @@
+rule:
+  meta:
+    name: authenticate message hash-based
```

Comment:
recognizable in the pseudocode here: https://en.wikipedia.org/wiki/HMAC

Reply:
how about just "authenticate HMAC"?

"hash-based" reads weird to me